core/src/main/resources/i18n/displayStrings.properties

```
@@ -608,6 +608,18 @@ portfolio.pending.step2_seller.waitPayment.msg=The deposit transaction has at le
 portfolio.pending.step2_seller.warn=The BTC buyer still has not done the {0} payment.\nYou need to wait until they have started the payment.\nIf the trade has not been completed on {1} the arbitrator will investigate.
 portfolio.pending.step2_seller.openForDispute=The BTC buyer has not started his payment!\nThe max. allowed period for the trade has elapsed.\nYou can wait longer and give the trading peer more time or contact the arbitrator for opening a dispute.
 
+tradeChat.chatWindowTitle=Chat window for trade with ID ''{0}''
+tradeChat.openChat=Open chat window
+tradeChat.rules=You can communicate with your trade peer to resolve potential problems with this trade. It is not mandatory to reply in \
+  the chat. If a trader violates the below rules, open a dispute with 'cmd + o' and report it to the arbitrator.\n\n\
+  Chat rules:\n\
+  \t● Do not send links to unknown URLs (risk of malware). Links to a block explorer are permitted.\n\
```

Should we also disallow link shorteners?

Even known URLs can be disguised to look convincing with weird TLDs.

I think all links should be disallowed, but maybe I'm paranoid.

---

Yes, good point with link shortener. I think it should be added. Maybe also recommend to use Tor browser as that is more safe with deactivated js etc...

---

Right. Actually beside links to the tx at a block-explorer no links would be needed. And those can be looked up by the other used with the tx id as well. So right maybe disallow any URLs?

---

> So right maybe disallow any URLs?

Yes, I would say so. 

```suggestion
  \t● URLs are not allowed. If you would like to send a link to a block explorer, for example, just send the transaction ID, and the other peer can look it up on their own.\n\
```